.github/workflows/clippy.yml

```diff
@@ -1,19 +1,14 @@
 on:
   push:
     branches: [ staging, trying, master ]
-  pull_request_target:
+  pull_request:
```

Comment:
What's the intention with changing this and removing the `pull_request_target` entries below? Won't it stop the clippy annotations working since the secret key becomes unavailable?

Reply:
oops, probably a rebase mistake. Fixed.